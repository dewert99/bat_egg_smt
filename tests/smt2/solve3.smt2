(declare-sort S1)
(declare-const a S1)
(declare-fun b () S1)
(declare-const x Bool)
(assert x)
(assert (= a b))
(check-sat)
(check-sat-assuming ((not (and x (= (a) (b))))))
(get-unsat-core)