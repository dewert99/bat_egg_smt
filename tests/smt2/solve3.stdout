sat
((a @S1_3)
 (b @S1_3)
 (x true))
(
 (define-fun a () S1 @S1_3)
 (define-fun b () S1 @S1_3)
 (define-fun x () Bool true)
)
unsat
((not (and x (= (a) (b)))))
